docs/app/pages/components/components-sections/tables/column-sorting/column-sorting.component.html

```diff
@@ -110,6 +193,12 @@
         Defines if a column can be toggled to the <code>NoSort</code> state. If set to <code>false</code> this will
         toggle between the <code>Ascending</code> and <code>Descending</code> states when changing the sort direction.
     </tr>
+    <tr uxd-api-property name="ascendingIcon" type="string" defaultValue="ascend">
+        Specify an icon name from the icon set for sorting in ascending order - <a routerLink="/css/icons" fragment="ux-icons">UX Aspects Icons</a>.
```

Comment:
```suggestion
        Specify an icon name from the <a routerLink="/css/icons" fragment="ux-icons">UX Aspects icon set</a> for sorting in ascending order.
```

Also this link does not work in the Micro Focus site because the route is different. In my previous review I recommended that you look at the icon component documentation section to understand how we do this properly - please check this out now.